src/lib/sql-client.ts

```diff
@@ -225,7 +225,13 @@ export class ExasolDriver implements IExasolDriver {
     sqlStatement: string,
     attributes?: Partial<Attributes> | undefined,
     getCancel?: CetCancelFunction | undefined,
-    responseType?: 'default' | 'raw'
+    responseType?: 'default' | 'raw',
+  ): Promise<QueryResult | SQLResponse<SQLQueriesResponse>>;
+  async query(
+    sqlStatement: string,
+    attributes?: Partial<Attributes> | undefined,
+    getCancel?: CetCancelFunction | undefined,
+    responseType?: 'default' | 'raw' | undefined,
   ): Promise<QueryResult | SQLResponse<SQLQueriesResponse>> {
```

Comment:
What is the reason for this change? I only see that `responseType` can now be `undefined`. But `?` already marks the parameter as optional, so `| undefined` should not be necessary.

Reply:
It's a typescript specific thing. I'll call you and show you. We can also discuss the separate config for the pool properties then.